src/database/entities/user.entity.ts

```diff
@@ -6,19 +6,19 @@ import {
     CreateDateColumn
 } from "typeorm";
 
-@Entity("users")
+@Entity("example.users")
 export class User {
     @PrimaryGeneratedColumn("uuid")
-    userId: string;
+    id: string;
 
-    @Column({ unique: true })
+    @Column()
     email: string;
 
-    @Column({ type: "timestamptz" })
+    @Column()
```

Comment:
No big deal, but I'm curious why you took out the type hints here.

Reply:
whoops, left over from when I was trying to get Mongo and Postgres working side by side